pom.xml

```diff
@@ -288,7 +289,11 @@
                 <groupId>io.delta</groupId>
                 <artifactId>delta-standalone_${scala.binary.version}</artifactId>
                 <version>${delta.standalone.version}</version>
-                <scope>test</scope>
```

Comment:
delta standalone is needed during runtime


<img width="1001" alt="image" src="https://github.com/user-attachments/assets/95901a56-85f0-4e6e-bc58-04d808e3b3ff" />